src/decode.h

```diff
@@ -90,7 +90,7 @@ enum op_field {
     )                                                  \
     /* RV32 Zicsr Standard Extension */                \
     IIF(RV32_HAS(Zicsr))(                              \
-        _(csrrw, 0, 4, 0, ENC(rs1, rd))                \
+        _(csrrw, 1, 4, 0, ENC(rs1, rd))                \
```

Comment:
> If `SYSTEM` configuration is set, the Zicsr should be set accordingly.

Zicsr is enabled by default if no configuration file or `ENABLE_Zicsr` parameter is used. So, would it be better notice this on README is good enough?

Reply:
> Zicsr is enabled by default if no configuration file or `ENABLE_Zicsr` parameter is used. So, would it be better notice this on README is good enough?

It would be good to document the two configuration options and clarify the dependency in build system.